mvp/api/tags_api.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: "3.0.0"
+servers:
+  - url: http://localhost:8080
+info:
+  version: 0.0.1
+  title: Nobis API
+tags:
+  - name: Tags controller
+
+paths:
+  /api/tags/add/{userId}:
```

Comment:
Does it override the tags or append them?

Reply:
PUT means "insert, replace if already exists" = )